rs/state_machine_tests/src/lib.rs

```diff
@@ -3574,6 +3574,21 @@ impl StateMachine {
             .get()
     }
 
+    /// Returns the reserved balance of the specified canister.
+    ///
+    /// # Panics
+    ///
+    /// This function panics if the specified canister does not exist.
+    pub fn reserved_balance(&self, canister_id: CanisterId) -> u128 {
```

Comment:
Why is a new function needed? Can't you just use the existing canister status endpoint (maybe together with get_controllers if you don't have a controller around)?

Reply:
good point, switched to using canister_status, thank you!